### PR TITLE
Update /kubernetes/managed page

### PR DESCRIPTION
**Pull request by solazio**
## Done

- update `/kubernetes/managed`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1707) and [copy doc](https://docs.google.com/document/d/1Qv379nYnod9e4jMrsdDND8D7mbPyqELi2lowjkfD-gU/edit)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1758

### CODE REVIEW

**Comment by anasereijo** [comment]
@solazio - [ ] one line of copy missing from the list below the hero

**Comment by solazio** [comment]
> @solazio - [ ] one line of copy missing from the list below the hero

@anasereijo well spotted! thanks

File: templates/kubernetes/managed.html [hunk 1]
```diff
@@ -3,32 +3,51 @@
...
+              href="/kubernetes">Kubernetes consulting
+              packages</a></li>
+          <li class="p-list__item is-ticked">Customise the architecture in iterations based on workloads</li>
+          <li class="p-list__item is-ticked">Enterprise SLA for the
+            infrastructure, upgrades included</li>
+          <li class="p-list__item is-ticked">Monitoring and log management
+            integration</li>
+        </ul>
+        <a
+          href="/kubernetes/contact-us?product=kubernetes-managed">Talk to us&nbsp;&rsaquo;</a>
```
Comment (sowasred2012) [inline templates/kubernetes/managed.html]: One question - should this trigger the modal?

Comment (solazio) [reply templates/kubernetes/managed.html]: @sowasred2012 good question. On the live page the `Talk to us` button triggers the modal. Perhaps this should do the same? @pmahnke what do you think?

**Pull request merged**